edgelet/edgelet-http/src/util/hyperwrap.rs

```diff
@@ -1,27 +1,42 @@
 // Copyright (c) Microsoft. All rights reserved.
 
+use crate::client::ClientImpl;
```

Comment:
Crate imports come after third-party crate imports with an empty line to separate them, like they were before this change.